intellij-plugin/build.gradle.kts

```diff
@@ -148,3 +149,47 @@ tasks.register("downloadMockJdk") {
 tasks.named("test").configure {
   dependsOn("downloadMockJdk")
 }
+
+tasks.register("generateUpdatePluginsXml") {
+  val filePath = "build/publications/updatePlugins.xml"
+  val pluginName = properties("pluginName")
+  val version = properties("VERSION_NAME")
+  outputs.file(filePath)
+  doLast {
+    // See https://plugins.jetbrains.com/docs/intellij/custom-plugin-repository.html
+    file(filePath).writeText(
+        """
+        <plugins>
+          <plugin
+              id="$pluginName"
+              url="https://s01.oss.sonatype.org/content/repositories/snapshots/com/apollographql/$pluginName/$version/$pluginName-$version.zip"
+              version="$version">
+            <idea-version since-build="${properties("pluginSinceBuild")}" until-build="${properties("pluginUntilBuild")}"/>
+            <name>Apollo GraphQL (Snapshot)</name>
+          </plugin>
+        </plugins>
+        """.trimIndent()
+    )
+  }
+}
+
+publishing {
+  repositories {
+    maven {
+      name = "ossSnapshots"
+      url = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
+      credentials {
+        username = System.getenv("SONATYPE_NEXUS_USERNAME")
+        password = System.getenv("SONATYPE_NEXUS_PASSWORD")
+      }
+    }
+  }
+
+  publications {
+    create<MavenPublication>("default") {
+      artifactId = properties("pluginName")
+      artifact(tasks.named("signPlugin"))
+      artifact(tasks.named("generateUpdatePluginsXml"))
```

Comment:
Shouldn't this publish the plugin .jar as well?

Reply:
It looks like this only publishes the .zip (the output of `signPlugin`), which is all that we need?

Reply:
Looks like it 👍